vllm/v1/utils.py

```diff
@@ -134,3 +141,48 @@ def shutdown(proc: multiprocessing.Process, input_path: str, output_path: str):
         socket_file = ipc_socket.replace("ipc://", "")
         if os and os.path.exists(socket_file):
             os.remove(socket_file)
+
+
+def bind_kv_cache(
```

Comment:
this is heavy code duplication, too.

Reply:
Though sharing the same funciton name, `bind_kv_cache` in v0 and v1 have very different behavior. v0 allocates kv cache as a list of tensor, and use `bind_kv_cache` to convert it to a dict for forward_context. v1 (after this pr) allocates kv cache as a dict of tensor, and use `bind_kv_cache` to convert it to a list for model runner. I prefer to keep them as two functions sharing the same function name.